data/formatters/linux.yaml

```diff
@@ -25,8 +25,12 @@ source: 'dpkg log file'
 ---
 type: 'basic'
 data_type: 'linux:locate_database:entry'
-message: 'Folder Paths: {paths}'
-short_message: '{paths}'
+message:
+- 'Folder Path: {path}'
+- 'Entries: {entries}'
+short_message:
+- '{paths}'
```

Comment:
`paths` => `path`

also keep `paths` for backwards compatibility

Reply:
Done.

Reply:
I've made some changes